students/tri_nguyen/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+# ------------------------------------------- #
+# Student:   Tri Nguyen
+# Instructor: Natasha Aleksandrova
+# Date:  10-Feb-2018
+# ------------------------------------------- #
+
+# Define functions
+
+
+def format_presentation(a_string):
+    ''' this function prints stuff for readability '''
+
+    print('\nTask {} returns the formatted string below.'. format(a_string))
+
+
+def task_one():
+    ''' this function will format a 4 element
+        tuple and produce
+        'file_002 :   123.46, 1.00e+04, 1.23e+04' '''
+
+    a_tup = (2, 123.4567, 10000, 12345.67)
+
+    file, floating, exp_one, exp_two = a_tup
```

Comment:
be careful about overwriting built-in functions ([file](https://docs.python.org/2/library/functions.html#file)), some editors will have special syntax highlighting.. but you can also see it in github highlighted orange